tests/test_index_reader.py

```diff
@@ -47,6 +47,20 @@ def setUp(self):
         self.searcher = search.SimpleSearcher(self.index_path)
         self.index_reader = index.IndexReader(self.index_path)
 
+        self.temp_folders = []
+        self.emoji_corpus_path = 'integrations/resources/sample_collection_json_emoji'
+
+    def test_doc_vector_emoji_test(self):
+        index_dir = 'temp_index'
+        self.temp_folders.append(index_dir)
+        cmd1 = f'python -m pyserini.index -collection JsonCollection -generator DefaultLuceneDocumentGenerator -threads 1 -input {self.emoji_corpus_path} -index {index_dir} -storeDocvectors'
+        _ = os.system(cmd1)
+        tempt_index_reader = index.IndexReader(index_dir)
+        doc_vector = tempt_index_reader.get_document_vector('CACM-3807')
+        self.assertAlmostEqual(doc_vector['🙂'], 1)
```

Comment:
why almost equal? this should be exact match, no?

Reply:
Sorry, it should be equal just fixed that